packages/pigeon/lib/objc_generator.dart

```diff
@@ -375,9 +375,13 @@ String _makeObjcSignature({
       _getSelectorComponents(func, lastArgName);
   final Iterable<String> argTypes = followedByOne(
     func.arguments.map((NamedType arg) {
-      final String nullable = func.isAsynchronous ? 'nullable ' : '';
+      // TODO(gaaclarke):  I'm not sure why the arguments are nullable if the
+      // function is asynchronous.  I believe this is an error but changing it
+      // is a breaking change.
+      final String nullable =
+          (arg.type.isNullable || func.isAsynchronous) ? '_Nullable' : '';
```

Comment:
Apple uses both in their headers.  I switched to `_Nullable` because it was required for the NSError handles so I just was going to keep everything similar.

Reply:
They may use both, but they encourage using `nullable` when possible ([ref](https://developer.apple.com/swift/blog/?id=25)) and our style guide [explicitly says to prefer `nullable`](https://google.github.io/styleguide/objcguide.html#nullability).

Reply:
This wasn't resolved. We shouldn't generate code that violates the style guide's explicit recommendations, in the name of consistency in the generator. Consistency was already a factor that was considered in the drafting of that recommendation.

Reply:
done